projects/event-lambdas/src/event-api-lambda/application.ts

```diff
@@ -99,8 +100,33 @@ export const createApp = (initConfig: AppConfig): express.Application => {
           });
           console.log(logJson);
 
+          const viewEvent: IUserTelemetryEvent = {
+            app: lambdaApp,
+            stage: lambdaStage,
+            type: "GUARDIAN_TOOL_ACCESSED",
+            value: true,
+            eventTime: new Date().toISOString(),
+            tags: {
+              email,
+              stage,
+              app,
+              path,
```

Comment:
We could consider namespacing these tags a bit better to reflect they are for the app being tracked, not the app doing the tracking  😵‍💫